core/lib/env_config/src/via_btc_sender.rs

```diff
@@ -7,3 +11,37 @@ impl FromEnv for ViaBtcSenderConfig {
         envy_load("via_btc_sender", "VIA_BTC_SENDER_")
     }
 }
+
+impl FromEnv for ViaL1Secrets {
+    fn from_env() -> anyhow::Result<Self> {
+        Ok(Self {
+            rpc_url: std::env::var("VIA_BTC_CLIENT_RPC_URL")
+                .context("VIA_BTC_CLIENT_RPC_URL")?
+                .parse()
+                .context("VIA_BTC_CLIENT_RPC_URL")?,
+            rpc_password: std::env::var("VIA_BTC_CLIENT_RPC_PASSWORD")
+                .context("VIA_BTC_CLIENT_RPC_PASSWORD")?
+                .parse()
+                .context("VIA_BTC_CLIENT_RPC_PASSWORD")?,
+            rpc_user: std::env::var("VIA_BTC_CLIENT_RPC_USER")
+                .context("VIA_BTC_CLIENT_RPC_USER")?
+                .parse()
+                .context("VIA_BTC_CLIENT_RPC_USER")?,
+        })
```

Comment:
Any reason not to use `envy_load` ?
```suggestion
        envy_load("via_l1_secrets", "VIA_BTC_CLIENT_")
```

Reply:
Nice catch!
Fixed: [commit](https://github.com/vianetwork/via-core/pull/158/commits/047d5ece398c1dec9349f3fb850ddac775917362)